pubsub/rocketmq/metadata.go

```diff
@@ -0,0 +1,84 @@
+// ------------------------------------------------------------
```

Comment:
Now we use apache license.

Reply:
change to: Licensed under the Apache License ?

Reply:
yes.

Reply:
> yes.

already changed